### PR TITLE
SOV-1010: zero collateral ratio healthbar component

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.atlassian.net/browse/SOV-1010

### CODE REVIEW

File: .changeset/gentle-adults-lie.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"frontend": patch
+"@sovryn/ui": patch
+---
+
+Sov 1010 zero collateral ratio healthbar component
```
Comment (creed-victor) [inline .changeset/gentle-adults-lie.md]:
```suggestion
SOV-1010: zero collateral ratio healthbar component
```

**Comment by pietro-maximoff** [comment]
@Rickk137 could you pls add some test cases?

File: apps/frontend/src/app/2_molecules/CollateralRatio/CollateralRatio.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          type={HeadingType.h3}
+        >
+          {value !== undefined ? `${value}%` : 'N/A'}
+        </Heading>
+      </div>
+      <HealthBar
+        start={90}
+        middleStart={110}
+        middleEnd={150}
+        end={250}
```
Comment (creed-victor) [inline apps/frontend/src/app/2_molecules/CollateralRatio/CollateralRatio.tsx:40]: Story asks healthbar slices to be configurable, i suggest you to have a prop which could accept array of settings like:
```
slices={[
  { from: 0, to: 90, className: 'bg-red' },
  { from: 91, to: 110, className: 'bg-yellow' },
  { from: 110, to: 150, className: 'bg-green' },
  ...
]}
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/2_molecules/CollateralRatio/CollateralRatio.tsx (hunk 2 above)
Comment (soulBit) [reply apps/frontend/src/app/2_molecules/CollateralRatio/CollateralRatio.tsx:40]: I think current implementation is fine (although I also like your suggestion), we don't need it to be that flexible at this stage

**Review by creed-victor — APPROVED** [review]

**Review by soulBit — APPROVED** [review]

**Pull request merged**